src/Files.App/Dialogs/CreateArchiveDialog.xaml.cs

```diff
@@ -125,33 +126,41 @@ public string FileName
 				}
 			}
 
-			private FileFormatItem fileFormat;
 			public FileFormatItem FileFormat
 			{
-				get => fileFormat;
+				get => FileFormats.First(format => format.Key == GeneralSettingsService.ArchiveFormatsOption);
```

Comment:
Is this null-safe?

Reply:
It should be

Reply:
I mean does C# code analysis show warning for it? We'd like to reduce numbers of possible null warnings. Sorry I can't access my PC

Reply:
No